view-creator-framework/src/test/resources/sample-view-generation-spec.conf

```diff
@@ -72,6 +73,33 @@ pinotRealtime = {
     storageType = "pinot_server"
     serverTag = "tier-for-hot-data"
   }]
+
+  starTreeIndexConfigs = [{
+    dimensionsSplitOrder = [
+      "tenant_id",
+      "start_time_millis"
+    ],
+    skipStarNodeCreationForDimensions = [],
```

Comment:
If we give `dimensionsSplitOrder`, do we need this `skipStarNodeCreationForDimensions`?

Reply:
both are two different configurations. `skipStarNodeCreationForDimensions` - can be enabled for dimensions where we don't need star node, say tenant_id in our case.

Reply:
pinot docs for reference
https://docs.pinot.apache.org/basics/indexing/star-tree-index#index-generation

I can explain if not clear.